modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/Header/hooks/useFilters.js

```diff
@@ -232,20 +226,22 @@ export default function useFilters(setFilterTerm, productName) {
 				filtersKeyType.push(
 					`maxClusterNodes le ${filters.keyType.value.maxNodes}`
 				);
-				showOnPrem = true;
+				if (filters.keyType.value.hasOnPremise) {
+					showOnPrem = true;
+				}
```

Comment:
```suggestion
				showOnPrem = filters.keyType.value.hasOnPremise;
```

Reply:
If filters.keyType.value.hasOnPremise is false, the showOnPrem variable can be false too?